src/ol/layer/VectorTile.js

```diff
@@ -127,12 +127,6 @@ class VectorTileLayer extends BaseVectorLayer {
      */
     this.un;
 
-    if (options.renderMode === VectorTileRenderType.IMAGE) {
-      //FIXME deprecated - remove this check in v7.
-      //eslint-disable-next-line
-      console.warn('renderMode: "image" is deprecated. Option ignored.')
-      options.renderMode = undefined;
-    }
```

Comment:
I wonder if the following assertion is also not relevant any more with this change: https://github.com/openlayers/openlayers/blob/ecadbc7d5c7d4eef2c31c6a711dcbfdaf7e5bf41/src/ol/layer/VectorTile.js#L131-L135

This feels a little unnecessary now, doesn't it?